bigchaindb/backend/rethinkdb/admin.py

```diff
@@ -0,0 +1,53 @@
+"""Database configuration functions."""
+import logging
+
+import rethinkdb as r
+
+from bigchaindb.backend import admin
+from bigchaindb.backend.schema import TABLES
+from bigchaindb.backend.exceptions import DatabaseOpFailedError
+from bigchaindb.backend.utils import module_dispatch_registrar
+from bigchaindb.backend.rethinkdb.connection import RethinkDBConnection
+
+logger = logging.getLogger(__name__)
+
+register_admin = module_dispatch_registrar(admin)
+
+
+@register_admin(RethinkDBConnection)
+def config(connection, *, table):
+    return connection.run(r.table(table).config())
+
+
+@register_admin(RethinkDBConnection)
+def reconfigure(connection, *, table, shards, replicas,
+                primary_replica_tag=None,  dry_run=False,
+                nonvoting_replica_tags=None):
+    params = {
+        'shards': shards,
+        'replicas': replicas,
+        'dry_run': dry_run,
+    }
+    if primary_replica_tag:
```

Comment:
This follows from https://www.rethinkdb.com/api/python/reconfigure/

It's possible to specify the number of replicas per server tags. Say someone would wish to control the number of replicas for the servers in a specific region (Central Europe) versus those in another region (Asia). When doing so, the primary server tag must be specified.

also related: https://www.rethinkdb.com/docs/sharding-and-replication/#server-tags